RecoEgamma/PhotonIdentification/plugins/PhotonMVANtuplizer.cc

```diff
@@ -21,11 +21,6 @@
 #include "FWCore/Framework/interface/Frameworkfwd.h"
 #include "FWCore/Framework/interface/one/EDAnalyzer.h"
 
-#include "FWCore/Framework/interface/Event.h"
```

Comment:
All the removed include's seem to be needed: why did you remove them?

Reply:
Sorry then I didn't quite get the idea of includes in CMSSW yet: I thought the includes every files should have is the minimum set on includes it needs to compile... But I understand now that this is the wrong philosophy, I'll add them back in the next commit!